frontend/src/common/types/http/http-options.type.ts

```diff
@@ -4,6 +4,7 @@ type HttpOptions = {
   method: HttpMethod;
   contentType: ContentType;
   payload: BodyInit | null;
+  hasAuth: boolean;
```

Comment:
not sure it should be required option, we have more routes with auth than without it. Don't want to pass value for every request